crates/rb-sys/src/stable_api/ruby_2_7.rs

```diff
@@ -96,6 +96,16 @@ impl StableApiDefinition for Definition {
         is_immediate || !test
     }
 
+    #[inline]
+    unsafe fn bignum_positive_p(&self, obj: VALUE) -> bool {
+        crate::rb_big_sign(obj) == 1
```

Comment:
let's re-implement the flag checks to avoid dylib dispatch (and so rustc can inline more). This will keep the performance on par with [the C macros](https://github.com/ruby/ruby/blob/bf225feb265af6d796721d3a7e2b6800d160d42d/internal/bignum.h#L171-L175)

Reply:
Forget about the truffle failures for now, we'll circle back on those and add a new definition file

Reply:
You mean like this? I had to whitelist `ruby_fl_types` in order to bindgen to generate. let me know if this is what you meant 😅

Reply:
yes exactly... suprised to see they were gone..... 🤔